content/docs/buildpack-author-guide/create-buildpack/adding-bill-of-materials.md

```diff
@@ -0,0 +1,171 @@
++++
+title="Adding Bill-of-Materials"
+weight=409
++++
+
+<!-- test:suite=create-buildpack;weight=9 -->
+
+One of the benefits of buildpacks is they can also populate the app image with metadata from the build process, allowing you to audit the app image for information like  what process types are available and what commands are associated with them, finding the run-image the app image was based on or even finding which buildpacks were used to create the app image.
+
+You can find all of this information using `pack` via its `inspect-image` command.
```

Comment:
Any thoughts on the introduction to BOM and pack inspect-image?

I paraphrased this from the rfcs